crates/node/Cargo.toml

```diff
@@ -9,24 +9,27 @@ license.workspace = true
 [dependencies]
 borsh.workspace = true
 camino = { workspace = true, features = ["serde1"] }
-clap = { workspace = true, features = ["derive"] }
+clap.workspace = true
 eyre.workspace = true
 futures-util = { workspace = true, features = ["io"] }
 libp2p.workspace = true
 owo-colors.workspace = true
 rand.workspace = true
+ring.workspace = true
```

Comment:
you should be able to save this dependency, we don't need to import `ring::aead` to determine the size of the statically-sized array